src/main/java/com/nextstep/lotto/store/LottoMaker.java

```diff
@@ -0,0 +1,29 @@
+package com.nextstep.lotto.store;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.nextstep.lotto.lotto.Lotto;
+import com.nextstep.lotto.lotto.LottoNumberGenerator;
+import com.nextstep.lotto.lotto.Lottos;
+
+public class LottoMaker {
+
+	private Lottos lottos;
```

Comment:
```suggestion
	private final LottoNumberGenerator lottoNumberGenerator;
```

LottoNumberGenerator를 인터페이스로 바꾸고 RandomLottoNumberGenerator라는 클래스를 추가해보면 어떨까요?